gateway/1.4/modules/ROOT/pages/_partials/task-reg-run-flex-gateway.adoc

```diff
@@ -77,9 +77,27 @@ to generate set of instructions that includes a command block with the registrat
 // tag::reg-command-intro[]
 To register a Flex Gateway with Anypoint Platform, you must enter the registration command and then the start command. Each command includes information specific to your Anypoint Platform instance and *must be updated before* executing. See <<Before You Begin>> for more information on how to find the information you will need.
 //end::reg-command-intro[]
+//
+// tag::k8s-connected-intro[]
+In this mode, the gateway is managed entirely through Anypoint Platform, where you can: 
+
+* Deploy APIs to your cluster.
+* Apply policies and other configurations.
+* Create all resources through Anypoint Platform. 
+
+Resource creation through `kubectl apply` is restricted to specific resource types.
```

Comment:
which resource types? 

I think we're contrasting this with the use of that command for local mode, so important to clarify.

Reply:
The following resources are allowed to be created in connected mode:
- Extension
- Service
- PolicyBinding of types `tls`, `tls-inbound` and `tls-outbound`

Reply:
The following resources are allowed to be created in connected mode:
- Configuration
- Secret
- Extension
- Service
- PolicyBinding of types `tls`, `tls-inbound` and `tls-outbound`

Reply:
@seblaz Can you create any of these resources listed here via Anypoint Platform?

Reply:
It's a bit complicated:
1. `Configuration` can't be created via Anypoint
2. `Secret` is only used for registration, so I think it's not important. 
3. `Extension` can be created via Anypoint with custom policies.
4. `Service` is created via Anypoint when deploying apis, but if a custom policy wants to make an http request it needs to create another `Service` which can't be created via Anypoint.
5. `PolicyBinding`s can be created via Anypoint

Reply:
thank you!